website/addons/dryad/settings/defaults.py

```diff
@@ -0,0 +1,32 @@
+CLIENT_ID = None
+CLIENT_SECRET = None
+
+SCOPE = ['repo']
+
+SET_PRIVACY = False
+
+#DRYAD OAI-PMH metadata harvesting settings
+#Note where requested, format as (date, prefix, dataset)
+DRYAD_OAI_IDENTIFY = "http://www.datadryad.org/oai/request?verb=Identify"
```

Comment:
Yup, you're right. The waterbutler folks had me remove the python Dryad module since the unit tests depend on using aiohttp very specifically. We can either remove these lines or do as WB does and remove the dependency.